commands/pro/pro.js

```diff
@@ -2,25 +2,23 @@ function fixEscapes(str) {
   return str.replace(/[^a-z0-9|]/ig, '\\$&');
 }
 
-
 let proLangRe;
 let pros;
-const TERMS_INCLUDE = './proTerms.js';
```

Comment:
There's an ESLint rule to prevent using string literals, I prefered to just move everything to inline, because it's still a constant expression.